demo/components_tab.cpp

```diff
@@ -33,6 +33,12 @@ ComponentsTab::ComponentsTab()
 bool ComponentsTab::onPrimaryButtonClicked(brls::View* view)
 {
     // TODO: actually do something here
+    primary_button->setText("Some basic text");
```

Comment:
What is the purpose of that? To test that setText and getText work?

While I like the intent, the demo is not the place for such a test. Actually... there is no place for such a test, as we currently don't have a testing suite with borealis.

I want one but I didn't find something that suited all my needs (aka hook into the tested app, run it for multiple frames and test what's going on _inside_). I think we would need a custom made one, which takes time and effort.

Reply:
Done